### PR TITLE
PORTIA-517 Add modernizr

### PR DESCRIPTION
**Pull request by sagelliv**
The ember addon that uses [modernizr](https://github.com/sir-dunxalot/ember-cli-modernizr) is buggy and does not have active development. I decided to download Modernizr optimized for the two features we are currently checking: `flexbox` and `mutationobserver`.

@ruairif 
@michalmo: I have implemented the module function as a promise but I am not sure if this is necessary.

### CODE REVIEW

**Comment by sagelliv** [comment]
* Removed `mutationobserver` as this is only used in the section used by splash.
* Using the full modernizr reference in the `vendor` directory as `bower install modernizr` copied the entire modernizr repo into `bower_components`. There might be a way for use to build a custom modernizr version depending on the feature we check, but the addon which has this implemented has many bugs for version 1.13+

File: portiaui/app/utils/browser-features.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+const { Promise } = Ember.RSVP;
+
+export default function hasBrowserFeatures() {
+    let features = Ember.A(['flexbox']);
```
Comment (michalmo) [inline portiaui/app/utils/browser-features.js]: We have the ember prototype extensions enabled, and use native arrays throughout

**Comment by sagelliv** [comment]
@michalmo -> Focused modernizr to the above features and removed `Ember.A`

**Comment by sagelliv** [comment]
@michalmo Wrap `Modernizr.on` with an ember promise for asynchronous feature resolution.

File: portiaui/app/utils/browser-features.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "csspointerevents", "opacity", "csstransforms", "csstransitions", "cssvhunit",
+        "classlist", "placeholder", "localstorage", "svgasimg", "datauri", "atobbtoa"
+    ];
+    let feature_promises = features.map((feature) => {
+        return new RSVP.Promise((resolve) => {
+            Modernizr.on(feature, (isFeatureActive) => { resolve(isFeatureActive); });
+        });
+    });
+
+    return RSVP.hash(feature_promises);
```
Comment (michalmo) [inline portiaui/app/utils/browser-features.js]: shouldn't this be RSVP.all ?

**Pull request merged**